appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -88,15 +90,8 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
         ];
     }
 
-    private getCommonSkus(): WebSiteManagementModels.SkuDescription[] {
+    private getAdvancedSkus(): WebSiteManagementModels.SkuDescription[] {
```

Comment:
Seems like something odd happened with the merge here and [here](https://github.com/microsoft/vscode-azuretools/pull/890/files?diff=split&w=1#diff-1487b1bd8f2d1b579960b612f89af39e94e334ac27dfca02ddfbcfcd6083e165R19)

Reply:
I copied the stuff which was different in master to my repo... didnt do a proper merge